lib/SIL/IR/SILFunctionType.cpp

```diff
@@ -4108,6 +4112,18 @@ TypeConverter::getLoweredFormalTypes(SILDeclRef constant,
       }
     }
 
+    // C++ operators that are implemented as non-static member functions get
+    // imported into Swift as static methods that have an additional
+    // parameter for the left-hand-side operand instead of the receiver object.
+    // These are inout parameters and don't get bridged.
```

Comment:
Could you add a TODO about possibly undoing it if we would stop using `inout`?

Reply:
Done